model/src/main/kotlin/config/Excludes.kt

```diff
@@ -60,30 +77,34 @@ data class Excludes(
      * excluded. If the [id] references a [Project] it checks if the [Project] is excluded and if the [id] does not
      * appear as a dependency of another non-excluded [Project].
      */
-    fun isExcluded(id: Identifier, analyzerResult: AnalyzerResult) =
-            analyzerResult.projects.find { it.id == id }?.let {
+    fun isExcluded(id: Identifier, ortResult: OrtResult) =
+            ortResult.analyzer?.result?.projects?.find { it.id == id }?.let {
                 // An excluded project could still be included as a dependency of another non-excluded project.
-                isProjectExcluded(it) && isPackageExcluded(id, analyzerResult)
-            } ?: isPackageExcluded(id, analyzerResult)
+                isProjectExcluded(it, ortResult) && isPackageExcluded(id, ortResult)
+            } ?: isPackageExcluded(id, ortResult)
 
     /**
-     * True if all occurrences of the package identified by [id] in the [analyzerResult] are excluded by this [Excludes]
+     * True if all occurrences of the package identified by [id] in the [ortResult] are excluded by this [Excludes]
      * configuration.
      */
-    fun isPackageExcluded(id: Identifier, analyzerResult: AnalyzerResult) =
-            analyzerResult.projects.all { project ->
-                isProjectExcluded(project) || project.scopes.all { scope ->
+    fun isPackageExcluded(id: Identifier, ortResult: OrtResult) =
+            ortResult.analyzer?.result?.projects?.all { project ->
+                isProjectExcluded(project, ortResult) || project.scopes.all { scope ->
                     isScopeExcluded(scope, project) || !scope.contains(id)
                 }
-            }
+            } ?: true
```

Comment:
Shouldn't the fallback value be `false` here (because if any of the objects is null, we cannot assume the package to be excluded)? Or it be written as ` == true` instead?

Reply:
Yes, you are right. The idea was that if there is no analyzer result the package is not contained in the result and therefore implicitly excluded, but that is not consistent with the other `is...Excluded` functions. Fixed.